CHANGELOG.MD

```diff
@@ -1,5 +1,21 @@
 # CHANGELOG
 
+## Unreleased
+	
+### Breaking
+[reff](https://github.com/juanfont/headscale/blob/a7874af3d0ba48913fa7e5fbffbb38b537bae7b3/CHANGELOG.md?plain=1#L74)
+- YAML files are no longer supported for headscale policy. [#1792](https://github.com/juanfont/OBheadscale/pull/1792)
+  - HuJSON is now the only supported format for policy.
+	
+### Fixed
```

Comment:
I think we can omit "Fixed" section as this is a new feature, rather than a fix of an old bug.